src/index.html

```diff
@@ -5,9 +5,52 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+
+      <div class="card__container">
+        <h2 class="card__container-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__container-product-code">
+          Product code: 195434
+        </p>
+
+        <div class="card__container-rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__container-reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__container-price">
+          <p class="card__container-price-title">Price:</p>
+          <p class="card__container-price-value">$2,199</p>
+        </div>
```

Comment:
the class names are not correct, please check this BEM article https://ru.bem.info/methodology/quick-start/

```suggestion
        <div class="card__container__price">
          <p class="card__container__price__title">Price:</p>
          <p class="card__container__price__value">$2,199</p>
        </div>
```